deploy/unifi.sh

```diff
@@ -72,29 +102,47 @@ unifi_deploy() {
     return 1
   fi
 
+  if [ "$_cloudkey_deploy" = "yes" ]; then
+    _info "Install Cloud Key certificate: $_cloudkey_certdir"
+    cp "$_cfullchain" "${_cloudkey_certdir}/cloudkey.crt"
+    cp "$_ckey" "${_cloudkey_certdir}/cloudkey.key"
+    (cd "$_cloudkey_certdir" && tar -cf cert.tar cloudkey.crt cloudkey.key unifi.keystore.jks)
```

Comment:
Do I need to manually check exit status of the `cp` and `tar` commands, or will any failures get caught and end the script? (There's already a preflight check that `$_cloudkey_certdir` is a writable directory, so I'm having trouble thinking of reasonable cases where these commands could fail.)